### ISSUE TITLE
[Global] Icon controls missing label

### ISSUE THREAD

**Issue by svinkle** [CovidShield/mobile#6]
## Issue summary

The clickable icon controls are missing a label. A label helps to describe what the control is for by describing its purpose of being. Navigating the app without the label descriptions will lead to a confusing user experience.

<img width="378" alt="Screen Shot 2020-05-22 at 11 56 55 AM" src="https://user-images.githubusercontent.com/1392632/82686161-6164b600-9c23-11ea-8cbe-7e01a770f74d.png">

#### TSX

```tsx
// TouchableIcon.tsx

const content = (
  <Box
    width={containerSize}
    height={containerSize}
    justifyContent="center"
    alignItems="center"
  >
    <Icon name={iconName} size={iconSize} />
  </Box>
);
```

## Steps to reproduce

1. Install the CovidShield `apk` on an Android device
2. Load the app
3. Go to the first screen with the "Covid Shield is ON" card
4. Enable [TalkBack screen reader](https://support.google.com/accessibility/android/answer/6007100?hl=en)
5. Swipe right to navigate to the icon control

### Expected

- Controls to be announce with a label description.

### Actual

- Label descriptions are missing.

## Recommendation

1. For each clickable component with an icon, include the [`accessibilityLabel`](https://reactnative.dev/docs/accessibility#accessibilitylabel-android-ios) prop.
2. Set an appropriate value based on the purpose of the control.

#### TSX

```tsx
// TouchableIcon.tsx

const content = (
  <Box
    width={containerSize}
    height={containerSize}
    justifyContent="center"
    alignItems="center"
  >
    <Icon name={iconName} size={iconSize} accessibilityLabel={label} />
  </Box>
);
```

## Specifications

- **Component:** CovidShield [Global]
- **WCAG Principle:** Understandable
- **WCAG SC:** 3.3.2 Labels or Instructions
- **Severity:** High
- **Effort:** Medium
